tools/bench_build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Stop on first error
+set -e
+
+# Move to the correct directory
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "${DIR}/.."
+
+echo dev profile: full build time
+cargo clean
+time cargo build -q
+
+echo;echo;
+echo dev profile: touch src/lib.rs
```

Comment:
It shouldn't do much. 
If you add from the end a `pub const FOO: usize = 42;` (and then remove it after the build completes) does it have different results?